fix command for running ssclj server

loomis opened the pull request:
Connected to #57

0xbase12 commented:
With the jar form of the command, I got this exception: 

[khaled@localhost jar]$ /usr/bin/java -Ddb.config.path=db.spec  -Dlogfile.path=log -jar target/SlipStreamCljResources-jar-2.15-SNAPSHOT-jar-with-dependencies.jar 8201
Exception in thread "main" java.lang.ExceptionInInitializerError
Caused by: java.lang.Exception: namespace 'com.sixsq.slipstream.ssclj.database.korma-helper' not found, compiling:(com/sixsq/slipstream/ssclj/db/database_binding.clj:2:1)

loomis commented:
Hmm, OK will look.

loomis commented:
Turns out that the jar option isn't very useful.  Because you cannot specify other elements on the classpath, there is no way to point to a `db.path` file.  The server logs that it cannot find the configuration file in this case, but unfortunately the service then fails with the class loading error.  (Admittedly not a very nice error.)

I've removed this option from the documentation.  I've also added a statement that the parent directory of the `db.spec` file must be on the classpath.
